addons/account/wizard/account_resequence_views.xml

```diff
@@ -26,14 +26,15 @@
             </form>
             </field>
         </record>
-
+    </data>
+    <data noupdate="1">
```

Comment:
The record now being in noupdate, an upgrade script is needed to force the update.

Reply:
Done https://github.com/odoo/upgrade/pull/2794